src/meta/ServerBasedSchemaManager.cpp

```diff
@@ -40,8 +41,12 @@ std::shared_ptr<const SchemaProviderIf>
 ServerBasedSchemaManager::getTagSchema(folly::StringPiece spaceName,
                                        folly::StringPiece tagName,
                                        SchemaVer ver) {
-    auto space = toGraphSpaceID(spaceName);
-    return getTagSchema(space, toTagID(space, tagName), ver);
+    auto spaceStatus = toGraphSpaceID(spaceName);
+    if (!spaceStatus.ok()) {
+        return std::shared_ptr<const SchemaProviderIf>();
+    }
+    auto spaceId = spaceStatus.value();
+    return getTagSchema(spaceId, toTagID(spaceId, tagName), ver);
```

Comment:
Verify that the return value of `toTagID`  is better? It might return -1 with no error.

Reply:
Usually, the cache can be found without having to judge the toTagID() every time. Sometimes we can't find it, we just do an extra search.